inspirehep/dojson/hep/fields/bd6xx.py

```diff
@@ -145,10 +146,16 @@ def _freekey_get_dict(elem):
 
     def _get_keyword_dict(key, elem):
         if _is_thesaurus(key) and elem.get('a'):
+            _schema = load_schema('hep')
```

Comment:
@jacquerie et al: this can be very heavy if not memoized. Is there a recommended way for memoization when not in an app context?